modules/pipeline/pipengine/reconciler/dbgc.go

```diff
@@ -239,6 +240,23 @@ func (r *Reconciler) DoDBGC(pipelineID uint64, gcOption apistructs.PipelineGCDBO
 		return nil
 	}
 
+	// If the pipeline is referenced by the definition and the record is GC
+	// the definition cannot see the execution record.
+	// There are two ways:
+	// 		one is to update the definition information contact binding
+	//	 	and the other is not to go to the GC pipeline record
+	// It is decided not to record this pipeline
+	if strings.TrimSpace(p.PipelineDefinitionID) != "" {
+		definition, err := r.dbClient.GetPipelineDefinitionByPipelineID(p.PipelineID)
```

Comment:
It is better to return the ‘has’ parameter or distinguish whether it exists or not.
Otherwise, a lot of non-existent logs will be printed.

Reply:
done